components/server-side-render/index.js

```diff
@@ -65,12 +66,8 @@ export class ServerSideRender extends Component {
 		} );
 	}
 
-	getQueryUrlFromObject( obj, prefix ) {
-		return map( obj, ( paramValue, paramName ) => {
-			const key = prefix ? prefix + '[' + paramName + ']' : paramName;
-			return isPlainObject( paramValue ) ? this.getQueryUrlFromObject( paramValue, key ) :
-				encodeURIComponent( key ) + '=' + encodeURIComponent( paramValue );
-		} ).join( '&' );
+	getQueryStringFromAttributes( attributes ) {
+		return isEmpty( attributes.attributes ) ? '' : '&' + httpBuildQuery( attributes );
```

Comment:
- `attributes` is the property of the function argument, not the argument itself ("object" was more appropriate)
 - `&` is beyond the responsibility of this function, it should be up to the caller to glue the query params to the url with `?` or `&`
I suggest ditching the function and moving the line to `fetch`